src/sentry/workflow_engine/endpoints/validators/base/detector.py

```diff
@@ -7,6 +7,7 @@
 from sentry import audit_log
 from sentry.api.fields.actor import ActorField
 from sentry.api.serializers.rest_framework import CamelSnakeSerializer
+from sentry.incidents.logic import enable_disable_detector
```

Comment:
let's move this method into this validator since it's a generic enable. then nit on the name to be `toggle` rather than `enable_disable`

that also breaks the final dependency on the incidents module here (ideally nothing is imported into workflow_engine from the incidents module)